packages/widgets/src/blocks/legacy-widget/edit/preview.js

```diff
@@ -78,6 +78,7 @@ export default function Preview( { idBase, instance, isVisible } ) {
 			) }
 			<div
 				className={ classnames(
+					cleanForSlug( 'widget_' + name ),
```

Comment:
Can we use a hyphen here instead?

Reply:
Also, same suggestion for the field to use here. `$id_base` should be a better field to use instead of `$name`.